README.md

```diff
@@ -102,7 +131,7 @@ Curious how this works, under the hood? [__Read the Medium post__](https://mediu
 
 ### Wrapping Element
 
-By default, FlipMove wraps the children you pass it in a `<div>`:
+By default, Flip Move wraps the children you pass it in a `<div>`:
```

Comment:
I noticed we were being inconsistent with `FlipMove` vs `Flip Move`, so I updated them all to `Flip Move`